src/components/Characters/Fox/FoxArtWork.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
```

Comment:
Please fix filename from FoxArt**W**ork to FoxArtwork

Reply:
Fixed in 45e27b4a0a4891eb695e744ec3664f5bfea0fa95

Reply:
Looks like 2d49b5e and 45e27b4 broke the build, but why?

Odd. Locally, `npm test` and `npm run build` works fine. Moreover, before those two commits, CI build passed, _even_ with the typo FoxArt**W**ork. Is there something I overlooked? Maybe it's a problem with _case-sensitive_? 🤔

Circel CI: https://circleci.com/gh/BuildIt-Poland/daily-routine/169

Reply:
First I fixed typo, build failed. I've added `.js` extension, but didn't work. After I bring back file name **with the typo** `../FoxArtWork` in this commit 4379880, build passed. 😲

Reply:
Maybe this - [Filenames being cached despite changing from lower case to upper case](https://discuss.circleci.com/t/filenames-being-cached-despite-changing-from-lower-case-to-upper-case/26627) will _put some light_ on the problem